images.toml

```diff
@@ -28,3 +28,4 @@ stripy = '36a3fec9586982c8226dc4bdabd5c9108adcb0c1'
 gangstr = '2.5.0'
 hipstr = 'v0.6.2'
 bgt = 'v1.0'
+expansionhunter_bw2 = '5.0.0-bw2'
```

Comment:
It would be best to alphabetise this alongside the other `expansionhunter` entry.